lib/azure_jwt_auth/authenticable.rb

```diff
@@ -4,6 +4,7 @@ module AzureJwtAuth
   AzureJwtAuth::NotAuthorized = Class.new(StandardError)
 
   module Authenticable
+
```

Comment:
This blank line doesn't follow the style of this gem. We don't add empty line at begining of modules.